sagemaker-python-sdk/tensorflow_script_mode_pipe_mode/tensorflow_script_mode_pipe_mode.ipynb

```diff
@@ -0,0 +1,186 @@
+    "\n",
+    "# Simple example data - a labeled vector.\n",
+    "features = {\n",
+    "    'data': tf.FixedLenFeature([], tf.string),\n",
+    "    'labels': tf.FixedLenFeature([], tf.int64),\n",
+    "}\n",
+    "\n",
+    "# A function to parse record bytes to a labeled vector record\n",
+    "def parse(record):\n",
+    "    parsed = tf.parse_single_example(record, features)\n",
+    "    return ({\n",
+    "        'data': tf.decode_raw(parsed['data'], tf.float64)\n",
+    "    }, parsed['labels'])\n",
+    "\n",
+    "# Construct a `PipeModeDataset` reading from a 'training' channel, using\n",
+    "# the TF Record encoding.\n",
+    "ds = `PipeModeDataset`(channel='training', record_format='TFRecord')\n",
+    "\n",
+    "# The `PipeModeDataset` is a TensorFlow Dataset and provides standard Dataset methods\n",
+    "ds = ds.repeat(20)\n",
+    "ds = ds.prefetch(10)\n",
+    "ds = ds.map(parse, num_parallel_calls=10)\n",
+    "ds = ds.batch(64)\n",
+    "\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Running training using the Python SDK\n",
+    "\n",
+    "We can use the SDK to run our local training script on SageMaker infrastructure.\n",
+    "\n",
+    "1. Pass the path to the pipemode.py file, which contains the functions for defining your estimator, to the sagemaker.TensorFlow init method.\n",
+    "2. Pass the S3 location that we uploaded our data to previously to the fit() method."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sagemaker.tensorflow import TensorFlow\n",
+    "\n",
+    "tensorflow = TensorFlow(entry_point='pipemode.py',\n",
+    "                        role=role,\n",
+    "                        framework_version='1.12.0',\n",
+    "                        input_mode='Pipe',\n",
+    "                        output_path=model_artifacts_location,\n",
+    "                        code_location=custom_code_upload_location,\n",
+    "                        train_instance_count=1,\n",
+    "                        py_version='py3',\n",
+    "                        train_instance_type='ml.c4.xlarge')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After we've created the SageMaker Python SDK TensorFlow object, we can call fit to launch TensorFlow training:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "import boto3\n",
+    "\n",
+    "# use the region-specific sample data bucket\n",
+    "region = boto3.Session().region_name\n",
+    "\n",
+    "train_data = 's3://sagemaker-sample-data-{}/tensorflow/pipe-mode/train'.format(region)\n",
+    "eval_data = 's3://sagemaker-sample-data-{}/tensorflow/pipe-mode/eval'.format(region)\n",
+    "\n",
+    "tensorflow.fit({'train':train_data, 'eval':eval_data})"
+   ]
+  }
+ ],
```

Comment:
I'd add a cell at the end to link to other notebooks to show what to do after training